### DISCUSSION TITLE
Sonidos

### DISCUSSION THREAD

**Discussion by gfazzito** [Godot-en-Espanol/Node-City#13] [Tareas]
### 1. _**Disparos**_

Viendo los modelos de gabo, el sonido del disparo está sumamente condicionado a:

- tamaño
- cantidad de cañones
- velocidad de disparo

Ejempo: Un cañon grande con tiempos de disparos largos debería tener un sonido pesado, lento y potente. Por el contrario, varios cañones chicos con disparos rápidos tendrán sonidos más agudos y livianos.
Estas variables se encuentran incluidas en la clasificación de enemigos (El tanque rápido tiene un cañon chico, un tanque pesado con 5 vidas tiene un cañon grande, etc).

### Impacto
Dependerán del material contra que impacta (metal, ladrillo, otro disparo, etc)

Colisión contra tanques: ver tipos de enemigos y salud de los mismos.
Colisión contra pared: ver tipos de materiales
Colisión contra otro disparo: ver tipos de disparo
Colisión contra bandera: sonido único
Colisión contra extremos del mapa: A definir

### Jugador vs enemigos
Se podrían hacer los disparos enemigos con características sonoras similares que contrasten con las del jugador. Esto da mayor información sobre los disparos

### Disparo viajando
En principio no seria necesario. No provee de información necesaria y puede llegar a ser molesto si hay muchos disparos en simultáneo

### Disparos en movimiento
Son los mismos a disparar estando quieto

### 2. _**Tanques**_

Hay sonidos cuando suceden las siguientes acciones:

- Movimiento: mismo sonido para cualquier dirección. Varía dependiendo el tipo de tanque y del terreno donde se encuentra.
- Recibir disparo: mismo sonido para cualquier disparo.
- Explotar: Varía dependiendo el tipo de tanque.

### Ideas

- Probar con dos tipos de sonidos para referir a las facciones (jugador vs enemigos)

**Comment by azagaya** [comment]
En cuanto al movimiento de los tanques, creo que serían 2 sonidos. Uno tipo de motor, que sea igual para todos, y otra capa que sea dependiente del terreno donde se mueve, no? Igual en principio con el sonido de motor safaría me parece.
